Add LONG_TEST flag to cvedb test test_01_nist_scrape (and maybe others)

This test periodically fails, I think due to rate limiting because we get a 403 forbidden response:

```
E           aiohttp.client_exceptions.ClientResponseError: 403, message='Forbidden', url=URL('https://nvd.nist.gov/feeds/json/cve/1.1/nvdcve-1.1-2002.meta')

/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/aiohttp/client_reqrep.py:1004: ClientResponseError

---------- coverage: platform linux, python 3.8.12-final-0 -----------
Coverage XML written to file coverage.xml

=========================== short test summary info ============================
FAILED test/test_cvedb.py::TestCVEDB::test_01_nist_scrape - aiohttp.client_ex...
============= 1 failed, 20 passed, 2 skipped in 173.85s (0:02:53) ==============
```

In the interests of being better users of NVD, I'm going to suggest that we move all of these tests to occur only in the LONG_TESTS phase so they don't run for every version of Python.  We'll probably be able to run them again once we get the API key set up, but I think in the interest of better using a shared resource we might want to limit how often we run these tests anyhow.

--- a/test/test_cvedb.py
+++ b/test/test_cvedb.py
@@ -4,6 +4,7 @@
 import datetime
 import shutil
 import tempfile
+from test.utils import LONG_TESTS
 
 import aiohttp
 import pytest
@@ -21,6 +22,9 @@ class TestCVEDB:
         shutil.rmtree(cls.cvedb.cachedir)
 
     @pytest.mark.asyncio
+    @pytest.mark.skipif(
+        LONG_TESTS() != 1, reason="Skipping NVD calls due to rate limits"
+    )
     async def test_00_getmeta(self):
         connector = aiohttp.TCPConnector(limit_per_host=19)
         async with aiohttp.ClientSession(
@@ -33,6 +37,9 @@ class TestCVEDB:
         assert "sha256" in meta
 
     @pytest.mark.asyncio
+    @pytest.mark.skipif(
+        LONG_TESTS() != 1, reason="Skipping NVD calls due to rate limits"
+    )
     async def test_01_nist_scrape(self):
         async with aiohttp.ClientSession(trust_env=True) as session:
             jsonshas = await self.cvedb.nist_scrape(session)
@@ -42,6 +49,9 @@ class TestCVEDB:
             )
 
     @pytest.mark.asyncio
+    @pytest.mark.skipif(
+        LONG_TESTS() != 1, reason="Skipping NVD calls due to rate limits"
+    )
     async def test_02_cache_update(self):
         async with aiohttp.ClientSession(trust_env=True) as session:
             jsonurl, meta = await self.cvedb.getmeta(
@@ -51,6 +61,9 @@ class TestCVEDB:
             await self.cvedb.cache_update(session, jsonurl, meta["sha256"])
 
     @pytest.mark.asyncio
+    @pytest.mark.skipif(
+        LONG_TESTS() != 1, reason="Skipping NVD calls due to rate limits"
+    )
     async def test_03_refresh(self):
         await self.cvedb.refresh()
         years = self.cvedb.nvd_years()

